engine/src/main/java/org/terasology/entitySystem/entity/EntityPool.java

```diff
@@ -53,6 +54,14 @@
      */
     EntityRef create();
 
+    /**
+     * Create different pools for each world and creates the global pool.
+     *
+     * @param gameManifest The game for which multiple pools will be needed.
+     * @return A new entity builder
+     */
+    EntityRef create(GameManifest gameManifest);
```

Comment:
why not add this to the EngineEntityManager interface?
Since the only real user is PojoEntityManager, which implements EngineEntityManager

Reply:
Name the method as something more intuitive, like `createWorldPools`